designs/static-capacity.md

```diff
@@ -0,0 +1,141 @@
+# Karpenter - Static Capacity
+
+## Background
+Karpenter currently operates as a dynamic cluster autoscaler, automatically adjusting node counts based on pending pod demand. However, several important use cases require maintaining a fixed set of nodes:
```

Comment:
Can we call out the specific use-cases that need manual provisioning?

Reply:
Added.